coding_style/README.adoc

```diff
@@ -226,6 +226,7 @@ Try editing files directly using other built-in modules (e.g. `ini_file`, `block
 If you are modifying more than a tiny number of lines or in a manner more than trivially complex, try leveraging the `template` module, instead.
 This will allow the entire structure of the file to be seen by later users and maintainers.
 The use of `lineinfile` should include a comment with justification.
+Alternatively, most configuration files have their own modules, such as community.general.ssh_config or community.general.nmcli.  Using these make code cleaner to read and ensure idempotence.
```

Comment:
```suggestion
Alternatively, most configuration files have their own modules, such as https://docs.ansible.com/ansible/latest/collections/community/general/ssh_config_module.html[community.general.ssh_config] or https://docs.ansible.com/ansible/latest/collections/community/general/nmcli_module.html[community.general.nmcli].
Using these make code cleaner to read and ensure idempotence.
```
One line per sentence.

Reply:
Yeah, but it bothers me that we are making recommendations that Red Hat Ansible customers won't be able to use . . .

Reply:
Ah, good point, sorry I approved before seeing your remark. Could we have other examples @bontreger ?